### ISSUE TITLE
'updateNoteFields

### ISSUE THREAD

**Issue by xx299x** [FooSoft/anki-connect#340]

**Comment by oakkitten** [comment]
I suppose you could call `notesInfo` to get the tags for notes, and then call `removeTags` to remove them, or call `replaceTags`. Not sure if `updateNoteFields` should support completely replacing tags.

**Comment by xx299x** [comment]
> I suppose you could call `notesInfo` to get the tags for notes, and then call `removeTags` to remove them, or call `replaceTags`. Not sure if `updateNoteFields` should support completely replacing tags.

Yes, you are right. It is a good idea, but it will spend a lot of time to call this `API` because it needs 3 operations.
